tests/rptest/tests/e2e_shadow_indexing_test.py

```diff
@@ -141,6 +141,10 @@ def setUp(self):
     def test_create_or_delete_topics_while_busy(self):
         self.logger.info(f"Environment: {os.environ}")
         if os.environ.get('BUILD_TYPE', None) == 'debug':
```

Comment:
nit: since you're editing this file, would you mind changing this line to `self.debug_mode` ?

Reply:
done